server/routes/referrals/referralsController.ts

```diff
@@ -22,10 +22,20 @@ import RarDaysView from './rarDaysView'
 import RarDaysPresenter from './rarDaysPresenter'
 import RarDaysForm from './rarDaysForm'
 import { FormValidationError } from '../../utils/formValidationError'
+import createInterventionsFormError from '../../utils/interventionsFormError'
 
 export default class ReferralsController {
   constructor(private readonly interventionsService: InterventionsService) {}
 
+  private validationErrorOrThrow(error: InterventionsServiceError) {
```

Comment:
Does this not need a return type?

Reply:
I don't think so - TypeScript does type inference for return types. But it might be helpful for readers here, I suppose. I've added one.